### PR TITLE
power: add eth008 power backend

### PR DESCRIPTION
**Pull request by JensKle**
**Description**
Adding a new power driver for robot electronics eth008 using HTTP-GET API.
Locally tested with labgrid-client power on/off/cycle.

**Checklist**
- [X] Documentation for the feature
- [X] Tests for the feature 

- [ ] The arguments and description in doc/configuration.rst have been updated

- [ ] Add a section on how to use the feature to doc/usage.rst

- [ ] Add a section on how to use the feature to doc/development.rst

- [ ] PR has been tested

- [ ] Man pages have been regenerated

### CODE REVIEW

**Comment by Bastian-Krause** [comment]
CI test errors are not your fault, #1277 fixes them.

**Review by Bastian-Krause — APPROVED** [review]
Looks good to me.

FYI: there is also https://github.com/pdudaemon/pdudaemon, a project that focuses only on PDUs. labgrid's PDUDaemonDriver interfaces with it. So it might be worth a thought to implement this rather over there. Let us know how you think about this.

Note: we should wait for #1277 and rebase this to make CI run this before merging.

File: labgrid/driver/power/eth008.py
Comment (Bastian-Krause) [inline labgrid/driver/power/eth008.py:42]:
```suggestion
    bool(int(value))
```
Maybe that's sufficient?

Comment (JensKle) [reply labgrid/driver/power/eth008.py:42]: Sorry what is wrong with my pullrequest? 
The Codecov Report fails, because of Indirect changes in labgrid/remote/coordinator.py.

What is the next step to get the pullrequest merged?

Comment (Emantor) [reply labgrid/driver/power/eth008.py:42]: Don't worry about codecov, it's not a required check for this PR to get merged. Next step for this PR is to maybe apply suggestion from @Bastian-Krause if it works.

Comment (JensKle) [reply labgrid/driver/power/eth008.py]: Changes applied.

**Review by Bastian-Krause — APPROVED** [review]

**Pull request merged**
